shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -34,6 +35,7 @@ @implementation FlutterViewController {
   fml::scoped_nsobject<FlutterDartProject> _dartProject;
   shell::ThreadHost _threadHost;
   std::unique_ptr<shell::Shell> _shell;
+  fml::WeakPtrFactory<FlutterViewController>* _weakFactory;
```

Comment:
We really try to avoid raw pointers. Please consider using a unique pointer. I realize the ObjC++ interop makes this a bit awkward.